sdk/mobile-center-analytics/src/test/java/com/microsoft/azure/mobile/analytics/AnalyticsTest.java

```diff
@@ -380,7 +383,7 @@ public void setEnabled() {
         assertTrue(Analytics.isEnabled().get());
 
         /* Disable. */
-        Analytics.setEnabled(false);
+        Analytics.setEnabled(false).get();
```

Comment:
Yeah it's just additional testing to make sure there is no deadlock or any issue by doing that as well.